include/dlr.h

```diff
@@ -47,8 +60,34 @@ typedef void* DLRModelHandle;
  success, -1 for error. Call DLRGetLastError() to get the error message.
  */
 DLR_DLL
-int CreateDLRModel(DLRModelHandle* handle, const char* model_path, int dev_type,
-                   int dev_id);
+int CreateDLRModel(DLRModelHandle* handle, const char* model_path, int dev_type, int dev_id);
+
+/*!
+ * \brief Creates a TVM DLR model.
+ * \param handle The pointer to save the model handle.
+ * \param graph String of loaded graph.json file
+ * \param lib_path Path to the TVM lib.so file
+ * \param params String of binary data loaded from tvm.params file
+ * \param params_len length of the params string
+ * \param dev_type Device type. Valid values are in the DLDeviceType enum in dlpack.h.
+ * \param dev_id Device ID.
+ * \return 0 for success, -1 for error. Call DLRGetLastError() to get the error message.
+ */
+DLR_DLL
+int CreateTVMModel(DLRModelHandle* handle, const char* graph, const char* lib_path,
```

Comment:
Can you use `size_t` type instead of `unsigned params_len`?

Add `#include <stddef.h>`

Reply:
done